src/driver/inceptionv3.cpp

```diff
@@ -35,2604 +36,1595 @@ migraphx::program inceptionv3(unsigned batch) // NOLINT(readability-function-siz
     migraphx::module_ref mmain = p.get_main_module();
     auto x_main_module_0       = mmain->add_literal(migraphx::abs(
         migraphx::generate_literal(migraphx::shape{migraphx::shape::float_type, {1}}, 0)));
-    auto x_0                   = mmain->add_parameter(
-        "0", migraphx::shape{migraphx::shape::float_type, {batch, 3, 299, 299}});
+    auto x_input               = mmain->add_parameter(
```

Comment:
Have you run each of these three driver models through `migraphx-driver verify` ?

Reply:
Looks like the models are good to go, no updates.